### PR TITLE
Create OWM-CHAT Plugin Submission

### PR DESCRIPTION
**Pull request by Cerisk**
Plugin takes the chat that npcs display in their dialog boxes and puts it in the chat box.

### CODE REVIEW

File: OWM-CHAT Plugin Submission [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/Cerisk/OWM-Chat.git
```
Comment (Nightfirecat) [inline OWM-CHAT Plugin Submission:1]: This file should not be added. The only one submitted should be `plugins/owm-chat`.

**Comment by Cerisk** [comment]
Also, I don't know why the checks were not successful so I thought it had to do with the changes I made to the gradle (putting in the information so I can create a shadowJar). I deleted those portions in the build.gradle so hopefully when it runs another check it can build it properly.

**Comment by Nightfirecat** [comment]
[Adding the shadowJar plugin is causing the build failure](https://github.com/Cerisk/OWM-Chat/blob/b413b9334ea88c9958bec5ccbe8391ffffe63166/build.gradle#L3), remove it and it'll work fine.

**Comment by Nightfirecat** [comment]
Your config group is still not really well namespaced. "User" isn't specific enough.

**Comment by Cerisk** [comment]
> Your config group is still not really well namespaced. "User" isn't specific enough.

Not sure exactly how I should change this to be more specific, so I just set it to be "userChatFilterConfig" since in the future I will have users be able to set which npcs / messages they will want filtered in their config.

**Comment by Nightfirecat** [comment]
It's often the same as your plugin name, so something like `owm-chat` is what I would've chosen. This is fine though.

**Pull request merged**